packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -392,7 +463,7 @@ class ToCiceroMarkVisitor {
                     let text = '';
                     for (const codeBlockSubNode of subNode.elements) {
                         if (codeBlockSubNode.name === 'w:r') {
-                            text = this.fetchFormattingProperties(codeBlockSubNode, TRANSFORMED_NODES.codeBlock);
+                            text = this.fetchFormattingProperties(codeBlockSubNode, TRANSFORMED_NODES.codeBlock, {});
```

Comment:
`nodeInformation` was `null` before. How does an empty object help?

Reply:
@algomaster99 
There were errors for codeblock test case, so I passed an empty object.

Reply:
I would suggest looking for the reason for the error instead of introducing such workarounds.

Reply:
@algomaster99 i know the reason and will explain in the meeting.